calculator.js

```diff
@@ -1,66 +1,66 @@
-// let lastValue = null;
-// let operator = null;
-// let lastButtonClicked = null;
+let lastValue = null;
```

Comment:
why did you touch this file?
if you want to uncomment thing to test it then you don't need to push it.
this is making very difficult for me to see your changes

Reply:
Sorry I forgot to re-comment them again
and I used to use:
 git add .